app.js

```diff
@@ -281,6 +237,58 @@ d.run(function () {
 			});
 		}],
 
+		webSocket: ['config', 'connect', 'logger', 'network', function (scope, cb) {
+			var webSocketConfig = {
+				workers: 1,
+				port: scope.config.port,
+				wsEngine: 'uws',
+				appName: 'lisk',
+				workerController: workersController.path,
+				perMessageDeflate: false,
+				secretKey: 'liskSecretKey',
+				pingInterval: 5000,
+				// How many milliseconds to wait without receiving a ping
+				// before closing the socket
+				pingTimeout: 60000,
+				// Maximum amount of milliseconds to wait before force-killing
+				// a process after it was passed a 'SIGTERM' or 'SIGUSR2' signal
+				processTermTimeout: 10000,
```

Comment:
yes, we are waiting `processTermTimeout` before calling `process.exit()` after receiving kill signal. New connections won't be accepted during that time.